aten/src/ATen/core/NamedRegistrations.cpp

```diff
@@ -12,7 +12,7 @@ TORCH_LIBRARY_IMPL(aten, Named, m) {
   m.impl("_bmm", CppFunction::makeFallthrough());
   m.impl("_bmm.out", CppFunction::makeFallthrough());
   m.impl("_cdist_forward", CppFunction::makeFallthrough());
-  m.impl("_fused_dropout", CppFunction::makeFallthrough());
+  m.impl("native_dropout", CppFunction::makeFallthrough());
```

Comment:
Note to myself: `_fused_dropout` was tested in test_cuda.py

Reply:
https://github.com/pytorch/pytorch/blob/6e3e57095cab409e5c346d57ee42fd89c0584708/test/test_cuda.py#L2936
https://github.com/pytorch/pytorch/blob/c458558334e4aef5107bb60807c76ea9a7078727/tools/code_analyzer/default_op_deps.yaml#L1048